docs/components/iam/deployment/configuration-variables.md

```diff
@@ -12,6 +12,13 @@ Environment variable | Description | Default value
 -----|-------------|--------------
 FEATURE_LDAP | Toggle LDAP support within IAM | false
 
+### Functionality
+
+Environment variable | Description | Default value
+-------|----------|-------
+ENFORCE_ACCESS_CONTROL | Controls the enforcing of permissions for the IAM component | false
+ENFORCE_HTTPS | Controls if the URLs supplied for client configurations have to be http/https. | true
```

Comment:
When set to false, the urls can be http or https, so it would be more correct to state that the urls must be `https` when set to true. Not sure on the wording